Decred Wallet/view_controller/AccountViewController.swift

```diff
@@ -16,7 +16,7 @@ protocol AccountDetailsCellProtocol {
 class AccountViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
     // MARK: - Properties
     
-    var myBalances: [AccountsData] = [AccountsData]()
+    var accountHeaderInfo: [AccountsData] = [AccountsData]()
```

Comment:
- There should be plural in this variable name as it is an array.
- `Account Header Info` does not make sense to me, is there a better name that can be used? If the data type is `AccountsData`, then `accounts` seem to be a fine variable name.
- Also, using `Data` in class names is redundant. Usually, we use `User` instead of `UserData`, `Car` instead of `CarData`, `Balance` instead of `BalanceData`. Can the `AccountsData` class/struct be renamed? Also, why is it plural `Accounts`Data? Does the properties of the class describe more than 1 account?

Reply:
changed to accountHeader